content/en/platform/corda/5.0-dev-preview-1/tutorials/building-cordapp/c5-basic-cordapp-flows.md

```diff
@@ -449,6 +449,178 @@ data class CreateBoardingTicketInitiator @JsonConstructor constructor(private va
 }
 ```
 
+## Write the `GiftVoucherToFriend` flow
+
+The `GiftVoucherToFriend` flow lets you to transfer the ownership of the issued voucher.
```

Comment:
@peterli-r3 Could you please add more details to this description? Why do I need the GiftVoucherToFriend flow?

Reply:
message you via Teams.

Reply:
updated